src/main/java/org/myrobotlab/service/Python.java

```diff
@@ -732,11 +732,14 @@ public void setLocalScriptDir(String path) {
     broadcastState();
   }
 
-  /*
-   * no longer needed
-   * 
-   * @Override public void startService() { super.startService(); }
-   */
+  @Override
+  public void startService() {
+    super.startService();
+    Map<String, ServiceInterface> services = Runtime.getLocalServices();
+    for (ServiceInterface s : services.values()) {

```

Comment:
this should probably be synchronized to make sure there's no race conditions.  a service could be started while in this loop and it would be missed.